go.mod

```diff
@@ -17,6 +17,8 @@ require (
 	k8s.io/klog/v2 v2.70.0
 )
 
+replace github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.16.0+incompatible // indirect
```

Comment:
Has this version been tested against a trivy scan (or some other security scanner?) I haven't tested to see if `v2.16.0+incompatible` passes the security scan. I know trivy claims the fix version is `v3.8.0`.

Reply:
Never mind. I see the fix in their source code was cherrypicked into their 2.16.0 version. Thanks!
https://github.com/emicklei/go-restful/blob/v2.16.0/cors_filter.go